resources/views/api/emails/voucher_history_email.blade.php

```diff
@@ -3,15 +3,15 @@
 @section('content')
 
     <section role="main">
-        <h1 style="text-align: center;">[xXx] Voucher Payment Records</h1>
+        <h1>Voucher Payment Records</h1>
```

Comment:
I am always unsure... is h1 for the site/ company name and h2 for page titles? No idea what happens in case of an email.